app/modules/detectx/scripts/install.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# detectx controller
+#CTL="${BASEURL}index.php?/module/detectx/"
+	# Set preference to include this file in the preflight check
```

Comment:
Please also drop an empty json so the client does not complain about the file not found